rocky/reports/report_types/safe_connections_report/report.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from typing import Any, Dict
+
+from django.utils.translation import gettext_lazy as _
+
+from octopoes.models import Reference
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import IPAddressV4, IPAddressV6
+from reports.report_types.definitions import Report
+
+CIPHER_FINDINGS = ["KAT-RECOMMENDATION-BAD-CIPHER", "KAT-MEDIUM-BAD-CIPHER", "KAT-CRITICAL-BAD-CIPHER"]
+
```

Comment:
For hostnames, should we also check for missing or expired certs?
KAT-NO-CERTIFICATE and KAT-CERTIFICATE-EXPIRED
possibly also KAT-CERTIFICATE-EXPIRING-SOON for soon to be expiring certs.

Reply:
That's only for web

Reply:
Certificates can be on any service that talks TLS (including mailservers, webservers, database servers). I think we can collect these findings on this subreport  and see where they fit in the aggregate report.

Reply:
In KAT certificates can only exist on websites.

Reply:
For now. Yes, and thats an oversight in itself. Im okay with moving this to another sprint to keep this one clean. However, lets make sure at least the cert issues show up in the aggregate report somewhere.

Reply:
https://github.com/minvws/nl-kat-coordination/issues/2081

Yes they will